File: src/Web/Web.Spa/Models/RewardedEmployeeNew.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;

namespace Belorusneft.Museum.Web.Spa.Models
{
    public class RewardedEmployeeNew
    {
        [Required(ErrorMessage = "Name is not assigned")]
        public string Name { get; set; }

        public string Position { get; set; }

        public DateTime DateBirth { get; set; }

        public DateTime? DateStart { get; set; }

        public DateTime? DateEnd { get; set; }

        public IEnumerable<RewardWithYear> Rewards { get; set; }

    }
}

File: src/Web/Web.Spa/ClientApp/src/app/models/rewardedemployeeNew.ts
import { RewardWithYear } from './rewardwithYear';

export class RewardedEmployeeNew {

    public id: string;

    public name: string;

    public position: string;

    public dateBirth: Date;

    public dateStart: Date;

    public dateEnd: Date;

    public rewards: RewardWithYear[];
}
